VERSIONING.rst

```diff
@@ -0,0 +1,44 @@
+*****************
+Versioning Policy
+*****************
+
+We use a three-part X.Y.Z (Major.Minor.Patch) versioning definition, as follows:
+
+* **X (Major)** version changes are significant and expected to break backwards compatibility.
+* **Y (Minor)** version changes are moderate changes. These include:
+
+  * Significant non-breaking feature additions.
+  * Any change to the version of a dependency.
```

Comment:
Do we need to call out dependency version changes? I'm not sure that really applies to S2N.